meeting-notes/2020-09-30.md

```diff
@@ -0,0 +1,58 @@
+# Falco Community Call
+
+[More Information](https://github.com/falcosecurity/community)
+[Zoom Link](https://zoom.us/my/cncffalcoproject)
+[Recording](https://youtu.be/mDEsExkpgLw)
+
+## 2020-09-30
+
+### MC
+
+- Leonardo Grasso [@leogr](https://github.com/leogr)
+ 
+### Who joined
+
+- Lorenzo Fontana ([@fntlnz](https://github.com/fntlnz))
+- Radu Andries ([admiral0](https://github.com/admiral0))
+- Thomas Labarussias ([Issif](https://github.com/Issif))
+- Spencer Krum ([nibalizer](https://github.com/nibalizer))
+
+### This week highlights
+
+- Falco 0.26.0 is out (?)
+    - [Release on GitHub](https://github.com/falcosecurity/falco/releases) is still draft
+    - [bug](https://github.com/falcosecurity/falco/issues/1418) in the default ruleset for 0.26.0 - so probably we need to release a 0.26.1 hotfix release
+    - We decided to publish the 0.26.0 live on the call and we will skip the announcement since we have to release 0.26.1 with the fix. We will do announcement and blog post all together for 0.26.1.
+    - Lorenzo and Leogr to release 0.26.1
```

Comment:
As decided internally, I'm going to release 0.26.1 with Lorenzo.
Doing it right now!